### PR TITLE
Changes after Week Zero

### PR DESCRIPTION
**Pull request by spaceghost24**
Should have readme changes, no bucket, collector indexer and drivetrain(?) changes, etc.

### CODE REVIEW

File: src/main/java/frc/robot/commands/TeleopIndexer.java [hunk 1]
```diff
@@ -23,20 +25,32 @@ public TeleopIndexer(Indexer indexer) {
...
-    indexer.setWheelVelocity(0.0);
+    indexer.setPower(0.0);
   }
 
   // Called every time the scheduler runs while the command is scheduled.
   @Override
   public void execute() {
-    indexer.setWheelVelocity(OI.operatorController.getLeftY()*0.5);
-    System.out.println(OI.operatorController.getLeftY()*0.5);
+    if (OI.operatorController.getRightBumper()||(OI.operatorController.getLeftTriggerAxis()>0.5)) {
```
Comment (r-wallis) [inline src/main/java/frc/robot/commands/TeleopIndexer.java:37]: comment on what's going on here might be helpful

File: src/main/java/frc/robot/commands/TeleopCollector.java [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+      collector.setLiftPosition(loweredCollectorPosition);
+      isCollectorDown=true;
+    } else if (OI.operatorController.getAButtonReleased()) {
+      collector.setLiftPosition(raisedCollectorPosition);
+      isCollectorDown=false;
+    }
+    drivetrain.getChassisSpeeds(chassisSpeeds);
+
+    if (isCollectorDown) {
+      if (OI.operatorController.getLeftTriggerAxis() > 0.5) {
```
Comment (r-wallis) [inline src/main/java/frc/robot/commands/TeleopCollector.java:61]: comment on what's going on here might be helpful

File: src/main/java/frc/robot/commands/DriveControls.java [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+  /** Creates a new DriveControls. */
+  public DriveControls(Drivetrain drivetrain_) {
+    drivetrain = drivetrain_;
+    addRequirements(drivetrain);
+  }
+
+  // Called when the command is initially scheduled.
+  @Override
+  public void initialize() {
+    SmartDashboard.putNumber("X", 0);
```
Comment (r-wallis) [inline src/main/java/frc/robot/commands/DriveControls.java]: should this be deleted now?

File: src/main/java/frc/robot/subsystems/Indexer.java [hunk 4]
```diff
@@ -4,15 +4,31 @@
 
 package frc.robot.subsystems;
 
+import com.ctre.phoenix.motorcontrol.ControlMode;
+import com.ctre.phoenix.motorcontrol.NeutralMode;
+import com.ctre.phoenix.motorcontrol.SupplyCurrentLimitConfiguration;
+import com.ctre.phoenix.motorcontrol.can.WPI_TalonSRX;
+// import com.ctre.phoenix.motorcontrol.can.WPI_TalonSRX;
```
Comment (r-wallis) [inline src/main/java/frc/robot/subsystems/Indexer.java]: remove extra commented out line

**Review by BenWertz — APPROVED** [review]

**Review by classygirl44 — APPROVED** [review]

File: simgui.json [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+      "window": {
+        "visible": true
+      }
+    }
+  },
+  "NTProvider": {
+    "types": {
+      "/FMSInfo": "FMSInfo",
+      "/LiveWindow/Bling": "Subsystem",
+      "/LiveWindow/Bucket": "Subsystem",
```
Comment (spaceghost24) [inline simgui.json]: Should this still be here?

Comment (BenWertz) [reply simgui.json]: I think that file's just from the VSCode simulation, the bucket isn't actually in the code anymore. I probably should add those files to .gitignore, though, so I'll do that.

Comment (spaceghost24) [reply simgui.json]: Sounds good.

**Pull request merged**